app/main.py

```diff
@@ -1,3 +1,30 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    DAYS_IN_WEEKS = 7
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        if days < 0:
+            return 0
+        else:
+            return days // OnlineCourse.DAYS_IN_WEEKS \
+                + bool(days % OnlineCourse.DAYS_IN_WEEKS)
```

Comment:
`bool(a % b)` isn't a Python style. I'd better to use `ceil`. It's more readable.